pages/1.13/release-notes/1.13.0/index.md

```diff
@@ -71,16 +65,55 @@ Features and capabilities that are introduced in DC/OS 1.13 are grouped by funct
 
     If you deploy DC/OS monitoring, you can leverage Mesosphere-provided Grafana-based dashboards. By installing and configuring the `dcos-monitoring` service, you can automatically create dashboards that enable you to quickly visualize the metrics that the `dcos-monitoring` package is collecting from the DC/OS cluster and DC/OS-hosted applications. For more information about using Grafana dashboards, see the [dashboard repository](https://github.com/dcos/grafana-dashboards).
 
+### Metrics
+DC/OS metrics are collected and managed through the Telegraf service. Telegraf provides an agent-based service that runs on each master and agent node in a DC/OS cluster. By default, Telegraf gathers metrics from all of the processes running on the same node, processes them, then sends the collected information to a central metrics database. 
+    
+With this release, you can use Telegraf to collect and forward information for the following additional DC/OS cluster components:
+- CockroachDB
+- ZooKeeper
+- Exhibitor
+- Marathon
+- Metronome
+
+You can also collect information about the operation and performance of the Telegraf process itself. This information is stored along with other metrics and available for reporting using the DC/OS monitoring service or third-party monitoring services. For information about the Telegraf plugin and the metrics that Telegraf collects about its own performance, see the documentation for the [Internal input plugin](https://github.com/influxdata/telegraf/tree/master/plugins/inputs/internal).
+
```

Comment:
```suggestion

You can expose metrics from tasks running on Mesos in Prometheus format. When a port configuration belonging to a task is labelled appropriately, the metrics endpoint on that port will be polled regularly over the lifetime of the task and metrics collected will be added to the Telegraf pipeline. 

For a detailed description of how to configure a task so that its metrics are collected in Prometheus format, see the 
[Prometheus input plugin](https://github.com/dcos/telegraf/tree/1.9.4-dcos/plugins/inputs/prometheus#mesos-service-discovery).

```

Reply:
very helpful addition. thanks for your input!